java-api-examples/run-offline-add-diacritics.sh

```
@@ -0,0 +1,37 @@
+  make
+  popd
+fi
+
+if [[ ! -f "./catt_eo_model_onnx/encoder.onnx" || ! -f "./catt_eo_model_onnx/decoder.onnx"  ]]; then
+  curl -SL -O https://github.com/abjadai/catt/releases/download/v2/eo_model_onnx.zip
+  unzip eo_model_onnx.zip -d catt_eo_model_onnx
+  rm eo_model_onnx.zip
+fi
+
+java \
+  -Djava.library.path=$PWD/../build/lib \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)